doc/content/modules/user-manual/pages/release-notes/2025.2.0.adoc

```diff
@@ -126,6 +126,7 @@ image::release-notes-diagrams-export-as-png.png[Export diagram as PNG, width=50%
 * `Concern Usage`
 * `Stakeholder Membership`
 
+- Package nodes can now be resized smaller than their default size
```

Comment:
Please specify the context, with something like, "In the _General View_ diagram _Package_ graphical nodes can now be resized smaller than their default size"